test/servinge2e/servicemesh_test.go

```diff
@@ -37,10 +37,11 @@ import (
 )
 
 type testCase struct {
-	name               string
-	labels             map[string]string // Ksvc labels
-	annotations        map[string]string // Revision template annotations
-	expectIstioSidecar bool              // Whether it is expected for the istio-proxy sidecar to be injected into the pod
+	name                string
+	labels              map[string]string // Ksvc labels
+	annotations         map[string]string // Revision template annotations
+	expectIstioSidecar  bool              // Whether it is expected for the istio-proxy sidecar to be injected into the pod
+	expectedMetricCount float64
```

Comment:
Let's please create an extra test for this. This has nothing to do with the Service to Service tests and will just make picking apart test results harder as the service-to-service tests might fail due to this.

Reply:
In the past we discussed that I should not create a separate test to save some time. I guess this kind of test which just sets the admin does not cost anything right?

Reply:
Well we just need to create a Knative Service and then check the stuff you're checking. That might cost a second or two more, but that's absolutely worth it here.

We wouldn't want to install Serving separately for it though, yeah. We should be good here with a separate test in the same suite.